lib/receiver/receiver_impl.cc

```diff
@@ -371,6 +372,8 @@ namespace gr
             break;
           }
 
+          d_bsic = (ncc << 3) | bcc;
```

Comment:
Instead of using bitwise operations to mix both ncc and bcc into this new member you could add `get_ncc()` and `get_bcc()` functions that return the values of `d_ncc` and `d_bcc` members respectively. This way IMHO the API would be a bit cleaner and easy to use as the user can get the values directly and don't have to use bitwise operations and read documentation/source to understand how the values are encoded.

If you prefer to keep it as it is than you need to document what `get_bsic()` function returns.

Reply:
Great, I will fix it soon.
Thank you!